service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/RespondentsUpdatedEventHandlerEmailTemplateTest.java

```diff
@@ -45,7 +53,9 @@
     CaseDetailsHelper.class,
     CaseUrlService.class,
     ObjectMapper.class,
-    FixedTimeConfiguration.class
+    FixedTimeConfiguration.class,
+    EmailNotificationHelper.class,
+    FeatureToggleService.class
```

Comment:
The number of classes here can be reduced if you use `@ContextConfiguration`, when I was last updating templates I changed a number of them to use this if you need an example. It means that we don't need to redefine things like the `ObjectMapper` and  `NotificationService`

Reply:
see the other template test you have done

Reply:
done